src/objectExplorer/objectExplorerService.ts

```diff
@@ -956,6 +956,7 @@ export class ObjectExplorerService {
             RefreshRequest.type,
             refreshParams,
         );
+        this._expandParamsToTreeNodeInfoMap.set(refreshParams, node);
```

Comment:
So reverting the original "fix" and this like is the proper fix instead?

Reply:
Yup, this is the proper fix for the issue.